src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Courses/CoursesDataManagementHub.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Threading;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+using Dfc.CourseDirectory.Core.DataManagement;
+using Dfc.CourseDirectory.Core.DataStore.Sql;
+using Dfc.CourseDirectory.Core.DataStore.Sql.Queries;
+using Dfc.CourseDirectory.Core.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.Extensions.Logging;
+
+namespace Dfc.CourseDirectory.WebV2.Features.DataManagement.Courses
+{
+    [Authorize]
+    public class CoursesDataManagementHub : Hub
```

Comment:
Don't think we need this yet (plus it's looking at VenueUploads here not CourseUploads)